docs/kubernetes/KubernetesIntegration.md

```diff
@@ -65,10 +69,6 @@ The `kubelet` can be configured to use Calico by starting it with the `--network
 On CoreOS machines, the Calico network plugin is installed to the `/etc/kubelet-plugins` directory, and so an additional flag is necessary to inform the `kubelet` of the location of the plugin.  On CoreOS, include the `--network-plugin-dir=/etc/kubelet-plugins` option when starting the `kubelet`.
 > Note: the `--network-plugin-dir=` option is only available in Kubernetes >= v1.1.0
```

Comment:
The command to run will vary based on deployment.  Calico only cares that this one option is included.